Resources/Locale/en-US/_Impstation/objectives/conditions/steal-target-groups.ftl

```diff
@@ -1,7 +1,57 @@
 # Thief single item
 
 steal-target-groups-recruiter-pen = recruiter pen
+steal-target-groups-punpunid = Pun Pun's ID card
+steal-target-groups-firesuit = atmos fire suit
+steal-target-groups-mopbucket = mop bucket
+steal-target-groups-shiningspring = shining spring
+steal-target-groups-seedextractor = seed extractor
+
+steal-target-groups-medtekcartridge = MedTek cartridge
+steal-target-groups-logprobecartridge = LogProbe cartridge
+steal-target-groups-astronavcartridge = AstroNav cartridge
+
+steal-target-groups-servicetechfab = service techfab machine board
+steal-target-groups-shipyardcomputercircuitboard = shipyard computer board
+steal-target-groups-commscomputercircuitboard = communications computer board
 
 # Thief Animal
 
-steal-target-groups-animal-pipi = Pipi
+steal-target-groups-finfin = Fin Fin
+steal-target-groups-finfinmeat = Fin Fin's Fin-flesh
+
+# Thief Structure
+
+steal-target-groups-generator = station's spare power generator
+steal-target-groups-aicore = AI
+steal-target-groups-particleaccelerator = particle accelerator
+
+steal-target-groups-ptech = PTech
+steal-target-groups-sectech = SecTech
+
+steal-target-groups-hydrobe = HyDrobe
+steal-target-groups-lawdrobe = LawDrobe
+steal-target-groups-secdrobe = SecDrobe
+steal-target-groups-bardrobe = BarDrobe
+steal-target-groups-pietyvend = PietyVend
+steal-target-groups-cargodrobe = CargoDrobe
+steal-target-groups-medidrobe = MediDrobe
+steal-target-groups-chemdrobe = ChemDrobe
+steal-target-groups-curadrobe = CuraDrobe
+steal-target-groups-atmosdrobe = AtmosDrobe
+steal-target-groups-engidrobe = EngiDrobe
+steal-target-groups-chefdrobe = ChefDrobe
+steal-target-groups-detdrobe = DetDrobe
+steal-target-groups-janidrobe = JaniDrobe
+steal-target-groups-scidrobe = SciDrobe
+steal-target-groups-robodrobe = RoboDrobe
+steal-target-groups-genedrobe = GeneDrobe
+steal-target-groups-virodrobe = ViroDrobe
+
+# Thief Group
+
+steal-target-groups-cigarettes = cigarettes
```

Comment:
with the way the objective descriptions are set up, this turns it into "I need to get 654 cigaretteses". should probably make all these group objectives singular as they are in the upstream thief objectives

Reply:
oh my god this was me fixing a different grammar error incorrectly lol

Reply:
oh this wont do that specifically for cigarettes because i gave cigarettes a custom objective description that doesn't call this locale string at all. you are correct on the plural but idk what to do about plushies being both singular and plural